api/app/api/v1/routes/booths/aggregate.py

```
@@ -0,0 +1,380 @@
+                "year": empty_breakdown,
+            },
+            "hardware_summary": {
+                "printers": {"online": 0, "error": 0, "offline": 0},
+                "payment_controllers": {"connected": 0, "disconnected": 0, "not_configured": 0},
+            },
+            "recent_alerts": [],
+            "alerts_count": 0,
+        }
+    
+    booth_ids = [b.id for b in booths]
+    booth_names = {b.id: b.name for b in booths}
```

Ruff (0.14.10)

Local variable `booth_names` is assigned to but never used

Remove assignment to unused variable `booth_names`

(F841)